SFFeatureFlag/Classes/Models/Condition.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Condition.swift
+//  SFFeatureFlag
+//
+//  Created by Maja Sapunova on 8.6.21.
+//
+
+import Foundation
+
+public struct Condition: Decodable {
```

Comment:
I'd suggest including small documentation for the models (e.g. purpose and specifics if any).

Reply:
Added comments for the properties in the models